sdk/keyvault/Azure.Security.KeyVault.Administration/src/KeyVaultPermision.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Security.KeyVault.Administration.Models
+{
+    ///<inheritdoc/>
+    [CodeGenModel("Permission")]
+    public partial class KeyVaultPermission
```

Comment:
This should be an extensible enum (see our .NET guidelines for the template).

Reply:
Deferring this until permissions are settable in the service.